validators/package_validator_linux.go

```diff
@@ -161,11 +162,12 @@ func (validator *packageValidator) validate(packageSpecs []PackageSpec, manager
 
 // getKernelRelease returns the kernel release of the local machine.
 func getKernelRelease() (string, error) {
-	output, err := exec.Command("uname", "-r").Output()
+	var utsname unix.Utsname
+	err := unix.Uname(&utsname)
 	if err != nil {
 		return "", fmt.Errorf("failed to get kernel release: %w", err)
 	}
-	return strings.TrimSpace(string(output)), nil
+	return strings.TrimSpace(unix.ByteSliceToString(utsname.Release[:])), nil
```

Comment:
this produces exactly the same output as `uname -r`. i confirmed locally.
but there are no real benefits for it, unless some distros don't have `uname` installed.
@tklauser have you seen such cases?

Reply:
@neolit123 I haven't seen such a case in the real world but I could imagine minimal locked-down distributions not shipping or blocking access to `uname`. Plus it's a single `uname` syscall instead of `fork`/`exec` which I guess doesn't really matter here but is still nice if it can be avoided. So it felt like an easy change to to, especially `x/sys` is already a dependency in kubernetes.

But feel free to disregard if this feels like churn.

Reply:
no, it seems good to me.
/approve

Reply:
https://github.com/kubernetes/system-validators/blob/85af5a8471fc6a9ce319eb637654432992f52c6c/validators/os_validator_unix.go#L42 

https://github.com/kubernetes/system-validators/blob/85af5a8471fc6a9ce319eb637654432992f52c6c/validators/types_unix.go#L99

Should these two be updated as well if so?

Reply:
@pacoxu thanks, I've updated these too.